src/partials/head.html

```diff
@@ -2,7 +2,11 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link href="css/main.scss" rel="stylesheet" />
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Be+Vietnam" />
+  <link href="https://fonts.googleapis.com/css?family=Be+Vietnam" rel="stylesheet" />
```

Comment:
I'd at `&display=swap` here too if you wish to use a fallback before the font gets loaded